### PR TITLE
Corrected all instances of 'Github' to 'GitHub'

### PR DESCRIPTION
**Pull request by HyperPolygon64**
Yes, I am this picky - this PR also replaces the single character in binary files, but that shouldn't cause any problems (I'd be very concerned if it did).

### CODE REVIEW

File: Docs/UpdateSupport.md [hunk 1]
```diff
@@ -66,11 +66,11 @@ Note for Serious Programmers:
 There exists a standalone NuGet Package Converter released as part of Tools.zip alongside Reloaded releases (starting with 1.6.2). If you are using CI/CD for your mods, feel free to use this as part of your automatic deployment process.
 ```
 
-### Github
+### GitHub
 
 **Support:** A package is considered as "supported" if the file below exists.
 
-Support for mod updates from Github Releases can be added by copying the `ReloadedGithubUpdater.json` file from the Launcher's `Template` folder the to mod folder.
+Support for mod updates from GitHub Releases can be added by copying the `ReloadedGitHubUpdater.json` file from the Launcher's `Template` folder the to mod folder.
```
Comment (Sewer56) [inline Docs/UpdateSupport.md:73]: Renaming this file is a breaking change.

File: Docs/UpdateSupport.md [hunk 2]
```diff
@@ -85,15 +85,15 @@ After copying, you should then edit the file to include the user/organization na
...
-To determine the version on Github, the Github service uses the `tag` assigned to each release. 
+To determine the version on GitHub, the GitHub service uses the `tag` assigned to each release. 
 
-If the Github version is higher than the local one, there is an update.
+If the GitHub version is higher than the local one, there is an update.
 
 ##### User Configuration
 
-Each mod with Github update support can be configured by the user. This can be done by editing the `ReloadedGithubUserConfig.json`  file, in the mod folder using a standard text editor. 
+Each mod with GitHub update support can be configured by the user. This can be done by editing the `ReloadedGitHubUserConfig.json`  file, in the mod folder using a standard text editor.
```
Comment (Sewer56) [inline Docs/UpdateSupport.md:96]: This one too.

File: Docs/UpdateSupport.md [hunk 3]
```diff
@@ -105,13 +105,13 @@ Each mod with Github update support can be configured by the user. This can be d
 
 If not present, this file will appear the next time the Reloaded Launcher is launched.
 
-**Note:** *ReloadedGithubUserConfig.json should not be included in any mod downloads!*
+**Note:** *ReloadedGitHubUserConfig.json should not be included in any mod downloads!*
```
Comment (Sewer56) [inline Docs/UpdateSupport.md:108]: Same as above.

File: Source/Reloaded.Mod.Loader.Update/Resolvers/GitHubLatestUpdateResolver.cs [hunk 4]
```diff
@@ -205,25 +205,25 @@ private async Task<IReadOnlyList<Release>> GetReleases(GithubConfig configuratio
             catch { return null; }
         }
 
-        /* Github Config. */
-        public class GithubUserConfig : JsonSerializable<GithubUserConfig>
+        /* GitHub Config. */
+        public class GitHubUserConfig : JsonSerializable<GitHubUserConfig>
         {
-            public const string     ConfigFileName = "ReloadedGithubUserConfig.json";
+            public const string     ConfigFileName = "ReloadedGitHubUserConfig.json";
```
Comment (Sewer56) [inline Source/Reloaded.Mod.Loader.Update/Resolvers/GitHubLatestUpdateResolver.cs]: Same as above.

File: Source/Reloaded.Mod.Loader.Update/Resolvers/GitHubLatestUpdateResolver.cs [hunk 5]
```diff
@@ -205,25 +205,25 @@ private async Task<IReadOnlyList<Release>> GetReleases(GithubConfig configuratio
...
             {
                 LastCheckTimestamp = timeStamp;
             }
         }
 
-        public class GithubConfig : JsonSerializable<GithubConfig>
+        public class GitHubConfig : JsonSerializable<GitHubConfig>
         {
-            public const string     ConfigFileName = "ReloadedGithubUpdater.json";
+            public const string     ConfigFileName = "ReloadedGitHubUpdater.json";
```
Comment (Sewer56) [inline Source/Reloaded.Mod.Loader.Update/Resolvers/GitHubLatestUpdateResolver.cs]: Same as above.

**Review by Sewer56 — CHANGES_REQUESTED** [review]
Check out this awesome review.

**Review by Sewer56 — CHANGES_REQUESTED** [review]
Still needs file name reverts in documentation.

**Review by Sewer56 — APPROVED** [review]
Stonks.

**Comment by HyperPolygon64** [comment]
PR also changes some instances of 'Gamebanana' to 'GameBanana' - didn't get the chance to commit those, they were pushed with another. :P

**Pull request merged**